app/actions/remote/channel.ts

```diff
@@ -264,11 +265,16 @@ export async function patchChannel(serverUrl: string, channelPatch: Partial<Chan
             });
             models.push(channelInfo);
         }
+
         const channel = await getChannelById(database, channelData.id);
         if (channel && (channel.displayName !== channelData.display_name || channel.type !== channelData.type)) {
             channel.prepareUpdate((v) => {
-                v.displayName = channelData.display_name;
+                // DM and GM display names cannot be patched and are formatted client-side; do not overwrite
+                if (channelData.type !== General.DM_CHANNEL && channelData.type !== General.GM_CHANNEL) {
+                    v.displayName = channelData.display_name;
+                }
                 v.type = channelData.type;
+                logInfo({channelData});
```

Comment:
Do we need this log? And in case we do, it should have a description

Reply:
Nope, forgot to remove and hadn't quite finished a final self-review (was focusing on a self-QA pass for the screenshots)